microsoft-edge/webdriver-chromium/index.md

```diff
@@ -64,37 +74,25 @@ To begin writing automated tests, make sure the Edge WebDriver version you insta
 
-To use WebDriver to automate Microsoft Edge, if you use Selenium, you must use Selenium 4, which has built-in support for Microsoft Edge (Chromium).
-
-To install Selenium 4, see [Install a Selenium library](https://www.selenium.dev/documentation/webdriver/getting_started/install_library/).  In case you need it, the nuget packages page is [Selenium.WebDriver](https://www.nuget.org/packages/Selenium.WebDriver).
-
-
-### Upgrading from Selenium 3
-
-To use WebDriver to automate Microsoft Edge, if you use Selenium, make sure you are using Selenium 4.  Selenium 3 is no longer supported.
+Selenium WebDriver is one popular implementation of a WebDriver framework.  Selenium is a cross-browser automation library that's language-agnostic and test-framework agnostic.  This article uses Selenium WebDriver only as an illustrative example, and because it fulfills most users' needs.  If you use Selenium, you must use Selenium 4; Selenium 3 is no longer supported.
 
-You must upgrade existing Selenium 3 tests to Selenium 4.  To learn more about upgrading to Selenium 4, see [Upgrade to Selenium 4](https://www.selenium.dev/documentation/webdriver/getting_started/upgrade_to_selenium_4/).
+To accomplish the same tasks using a WebDriver testing framework other than Selenium WebDriver, consult the official documentation for your framework of choice.
 
-If you're using [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) to add Microsoft Edge (Chromium) support to your Selenium 3 browser tests, update your tests as follows:
 
-1. Remove Selenium Tools for Microsoft Edge from your project.  You don't need to use Selenium Tools for Microsoft Edge with Selenium 4, because Selenium 4 already has built-in support for Microsoft Edge (Chromium).
+<!-- ------------------------------ -->
+#### Using Selenium WebDriver
 
-1. Update your tests to use the built-in `EdgeDriver` and related classes that Selenium 4 provides instead.
+One possible WebDriver framework you can use is Selenium WebDriver, which is part of the Selenium suite of tools.
 
-1. Remove all usages of the `EdgeOptions.UseChromium` property.  This property no longer exists in Selenium 4, because Selenium 4 supports only Microsoft Edge (Chromium).
+Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript.  To run Selenium 4 tests, Python 3 is required (Python 2.7 is not supported).
 
-<!-- decorative hr; not a tab-end indicator: -->
+To use WebDriver to automate Microsoft Edge, if you use Selenium, you must use Selenium 4, which has built-in support for Microsoft Edge (Chromium).
 
----
+To install Selenium 4, see [Install a Selenium library](https://www.selenium.dev/documentation/webdriver/getting_started/install_library/).  In case you need it, the NuGet packages page is [Selenium.WebDriver](https://www.nuget.org/packages/Selenium.WebDriver).
```

Comment:
```suggestion
For detailed instructions on installing Selenium for your preferred language and development environment, see the Selenium project's official documentation on [Installing a Selenium library](https://www.selenium.dev/documentation/webdriver/getting_started/install_library/).
```

Note: Making it more clear that we are intentionally linking to external docs, and that these are the authoritative source for installing Selenium. Also removing the NuGet link which was a cause of confusion in the issue that prompted this PR. The linked Selenium docs has the NuGet info for those who are interested.

Reply:
done locally, slightly streamlined / low key